src/runtime/render.rs

```diff
@@ -73,20 +70,19 @@ impl Renderer {
         Self::default()
     }
 
-    /// Renders the animation at a given time into the specified scene builder.
+    /// Renders the animation at a given time into the specified scene.
     pub fn render(
         &mut self,
         animation: &Composition,
-        time: f32,
+        frame: f32,
         transform: Affine,
         alpha: f32,
         sink: &mut impl RenderSink,
     ) {
-        let frame = animation.frame_for_time(time);
         self.render_frame(animation, frame, transform, alpha, sink);
     }
```

Comment:
Yes, I found there to be significant benefit to choosing a frame to render instead of seconds.

I made this change when I implemented the dotLottie spec - which is built for interactivity. It was much easier to control a playhead (frame) rather than seconds, because of things like slow motion, bounce play mode, etc.

Also, in the seconds implementation, it would just wrap (modulo) the animation length, which didn't I found to be very unclear and hard to track the frame.

I removed the modulo wrapping guard - now if you render frame 51 on a 50 frame animation, you will see nothing because no layers apply to frame 51.

In general I do see there is value in render_t or something with modulo to expose the previous behavior, though.

Thoughts?

Reply:
Completely agree that we eventually want more sophisticated control over time to frame mapping supporting different looping behaviors. I think it makes sense to punt that to a future PR where we can focus on the details and get it right. For this PR, I’d just prefer not to have two methods that do the same thing so let’s remove one of them. Which name to keep is up to you.

Reply:
I consolidated these, choosing to keep frame instead of time.